monitor/utils/events.js

```diff
@@ -96,11 +97,40 @@ async function main(mode) {
       }
     })).map(normalizeEvent)
 
+    const foreignBridgeSwapped = new web3Foreign.eth.Contract([TOKENS_SWAPPED_EVENT_ABI], COMMON_FOREIGN_BRIDGE_ADDRESS)
+    const tokensSwappedEvents = (await getPastEvents(foreignBridgeSwapped, {
+      event: 'TokensSwapped',
```

Comment:
Will it be more safe to also check that the emitter of `TokensSwapped` is the bridge contract address?

Reply:
Yes, applied the change https://github.com/poanetwork/tokenbridge/pull/224/commits/a52b23cc0f4aa7e58bda9cf034d9059cf2192c67